topology.yaml

```diff
@@ -77,3 +77,13 @@ start args:
     --spark2-version:
         help: If specified, the version of Spark2 to install from archive.cloudera.com
         metavar: ver
+    --ssl-encryption:
+        action: store_true
+        help: If specified, enable SSL encryption for one or more cluster services
+    --ssl-encryption-authentication:
```

Comment:
Do we maybe wanna combine these in someway? What happens if someone were to pass both `--ssl-encryption` and `--ssl-encryption-authentication`? Perhaps make an argument called `--ssl` and allow choices of `encryption` and `encryption_authentication` to avoid the ambiguity? Argparse makes this straightforward if I'm not mistaken.

Reply:
Done.